README.md

```diff
@@ -240,11 +239,13 @@ Add the mParticle snippet to your `web/index.html` file as high as possible on t
 ```
 For more help, see the [full Web set up docs](https://docs.mparticle.com/developers/sdk/web/getting-started/#create-an-input).
 
-# Usage
+## Usage
```

Comment:
i see some are ##, ###, and #### in this PR...is there a difference between the amount?

Reply:
that's markdown notation for heads. `#` is the largest font/size down to `#####` which is just slightly larger than regular text :)

https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax